### PR TITLE
Suggestion for agree flag test.

### PR DESCRIPTION
**Pull request by ExternalReality**
Suggestion for how to test the agree flag.

### CODE REVIEW

**Comment by ExternalReality** [comment]
[follow on to PR 8847](https://trello.com/c/w7pdaDLH/321-follow-on-to-pr-8847)

File: cmd/juju/machine/upgradeseries_test.go [hunk 1]
```diff
@@ -144,7 +109,23 @@ func (s *UpgradeSeriesSuite) TestCompleteCommandDoesNotAcceptSeries(c *gc.C) {
...
+}
+
+func (s *UpgradeSeriesSuite) TestPrepareCommandShouldAcceptAgreeAndNotPrompt(c *gc.C) {
+	err := s.runUpgradeSeriesCommandWithConfirmation(c, "n", machine.PrepareCommand, machineArg, seriesArg, "--agree")
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(s.ctx.Stdout.(*bytes.Buffer).String(), gc.Equals, ``)
+}
+
+func stubRemoteAPI(c *gc.C) *mocks.MockUpgradeMachineSeriesAPI {
+	mockController := gomock.NewController(c)
```
Comment (manadart) [inline cmd/juju/machine/upgradeseries_test.go]: Even though this is for a stub and the only expectations use ```AnyTimes()```, I would still call ```Finish()``` on the controller at some point to ensure you did not get calls had no accompanying expectation (I *think* unexpected methods with no return would not break tests here).

I generally either return the controller from, or pass one into such methods.

**Comment by ExternalReality** [comment]
!!build!!

**Comment by hmlanigan** [comment]
LGTM, curious about the outcome from mandart's comment.

File: cmd/juju/machine/upgradeseries_test.go (hunk 1 above)
Comment (ExternalReality) [reply cmd/juju/machine/upgradeseries_test.go]: I don't agree

**Pull request closed without merging**